src/neo-vm/ExecutionEngine.cs

```diff
@@ -1202,6 +1202,14 @@ private void ExecuteInstruction()
                         x.Clear();
                         break;
                     }
+                case OpCode.POP:
+                    {
+                        VMArray x = Pop<VMArray>();
+                        int index = x.Count - 1;
```

Comment:
```suggestion
                        int index = x.Count - 1;
                        if (index < 0) throw new IndexOutOfRangeException();
```

Reply:
what is index exceeds length of `x`

Reply:
`RemoveAt()` will throw the same exception.